pandas/tests/arrays/test_datetimes.py

```diff
@@ -760,7 +760,7 @@ def test_date_range_frequency_M_Q_Y_raises(self, freq):
         with pytest.raises(ValueError, match=msg):
             pd.date_range("1/1/2000", periods=4, freq=freq)
 
-    @pytest.mark.parametrize("freq_depr", ["2MIN", "2mS", "2Us"])
+    @pytest.mark.parametrize("freq_depr", ["2MIN", "2nS", "2Us"])
```

Comment:
With this PR, `"2mS"` would just raise a hard error. I think that's OK tbh, anyone using 'ms' case-insentively was already playing with fire